src/Components/MessageList.js

```diff
@@ -0,0 +1,70 @@
+import React, { useCallback, useRef, useState, useEffect } from "react";
+import faker from "faker";
+import { Button, TextField } from "@mui/material";
+import { Message } from "../Message.js";
+
+// const initMess = [
+//   { author: "man", text: "mess1", id: faker.datatype.uuid() },
+//   { author: "man", text: "mess2", id: faker.datatype.uuid() },
+//   { author: "man", text: "mess3", id: faker.datatype.uuid() },
+// ];
+
+export const MessageList = () => {
+  const [messages, setMessages] = useState([]);
+  const [value, setValue] = useState("");
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    if (messages[messages.length - 1]?.author === "man") {
+      setTimeout(() => {
```

Comment:
https://stackoverflow.com/questions/53090432/react-hooks-right-way-to-clear-timeouts-and-intervals